plugins/turing/runner/experiment_runner.go

```diff
@@ -109,13 +134,60 @@ func (er *experimentRunner) GetTreatmentForRequest(
 		return nil, fmt.Errorf("Error marshalling the treatment config: %s", err.Error())
 	}
 
+	statusCode = http.StatusOK
+
 	return &runner.Treatment{
 		ExperimentName: filteredExperiment.Name,
 		Name:           selectedTreatment.Name,
 		Config:         rawConfig,
 	}, nil
 }
 
+func (er *experimentRunner) RegisterMetrics(
+	collector metrics.Collector,
+	metricsRegistrationHelper runner.MetricsRegistrationHelper,
+) error {
+	er.appContext.MetricService.ReplacePrometheusCollector(collector)
+	err := metricsRegistrationHelper.Register([]routerMetrics.Metric{
+		{
+			Name:        string(instrumentation.FetchTreatmentRequestDurationMs),
+			Type:        routerMetrics.HistogramMetricType,
+			Description: instrumentation.FetchTreatmentRequestDurationMsHelpString,
+			Buckets:     instrumentation.RequestLatencyBuckets,
+			Labels:      instrumentation.FetchTreatmentRequestDurationMsLabels,
+		},
+		{
+			Name:        string(instrumentation.ExperimentLookupDurationMs),
+			Type:        routerMetrics.HistogramMetricType,
+			Description: instrumentation.ExperimentLookupDurationMsHelpString,
+			Buckets:     instrumentation.RequestLatencyBuckets,
+			Labels:      instrumentation.ExperimentLookupDurationMsLabels,
+		},
+		{
+			Name:        string(instrumentation.FetchTreatmentRequestCount),
+			Type:        routerMetrics.CounterMetricType,
+			Description: "Counter for no. of Fetch Treatment requests with matching experiments",
```

Comment:
I guess this description and the next one can also be re-used from the `instrumentation` package?

Reply:
Yep that's true, I've updated it to reuse the description from that package.